.github/workflows/docs.yml

```diff
@@ -0,0 +1,49 @@
+name: Docs
+
+on:
+  release:
+    types: 
+      - created
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+
+      - name: Checkout docs
+        uses: actions/checkout@v3
+        with:
+          repository: dappnode/DAppNodeDocs
+
+      - name: Set up dappnodesdk
+        run: npm install -g @dappnode/dappnodesdk@${{ github.event.release.tag_name }}
+
+      - name: Download script from sdk
+        run: wget https://raw.githubusercontent.com/dappnode/DAppNodeSDK/master/.github/workflows/scripts/docs.cjs -O docs.cjs
+
+      - name: command_test
+        run: |
+          node docs.cjs
+          rm docs.cjs
+      
+      - name: Commit to docs
+        run: |
+          git add docs/dev/sdk/commands.md
+          git config --global user.email "actions@github.com"
+          git config --global user.name "GitHub Actions"
+          git commit -m "Updated docs for new SDK release"
+
+      - name: Pull request to docs
+        uses: peter-evans/create-pull-request@v5
```

Comment:
I dont like the idea of having a repository A making PRs on a repository B unless is strictly necessary to do so. Instead I would follow a different approach: 

The dappnodeSDK should contain in its releases the necessary assets for the documentation repository to document the dappnodeSDK. The dappnode documentation could have as a dependency the dappnodeSDK, using the exports `commands.js` and files references (`manifest-schema.json`, `setupWizard-schema.json`) to create the sdk documentation. For the Github actions there should be used a different approach as long as they are `.yml` files. **Consider parsing and exporting the github actions yaml files in `.json` format so they can be exported and used as a javascript dependency too in the dappnode documentation repo**